include/wx/qt/private/winevent.h

```diff
@@ -458,4 +458,23 @@ class wxQtEventSignalHandler : public Widget, public wxQtSignalHandler< Handler
     }
 };
 
+// Used by wxTextEntry derived classes to toggle the dialog's default button
+// off and on. see src/qt/textentry.cpp for the reason.
+#define wxHANDLE_DIALOG_DEFAULT_BUTTON(Widget)              \
```

Comment:
Using macro for this is rather ugly, could we use just use a function instead, i.e. `wxQtToggleDefaultButtonOnFocusEvent()`, and override the methods explicitly where this needs to be done? This would be much more clear and IMO the code reuse of the macro is not worth it in this case.